src/Nethermind/Nethermind.Evm/Tracing/GethStyle/GethTxTraceEntry.cs

```diff
@@ -15,25 +19,30 @@ public GethTxTraceEntry()
             Memory = new List<string>();
         }
 
+        [JsonConverter(typeof(LongRawJsonConverter))]
         public long Pc { get; set; }
 
-        [JsonProperty(PropertyName = "op")]
+        [JsonPropertyName("op")]
         public string? Operation { get; set; }
 
+        [JsonConverter(typeof(LongRawJsonConverter))]
```

Comment:
Downside of this approach is that we need to mark each property everywhere, instead of just relying on proper type conversion